.github/workflows/java-8-builds.yml

```diff
@@ -0,0 +1,30 @@
+name: Java 8 CI (MC 1.13-1.16)
+
+on: [push, pull_request]
+
+jobs:
+    build:
+        if: "! contains(toJSON(github.event.commits.*.message), '[ci skip]')"
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                  submodules: recursive
+            - name: Set up JDK 17
+              uses: actions/setup-java@v2
+              with:
+                  java-version: '17'
+                  distribution: 'adopt'
+                  cache: gradle
+            - name: Grant execute permission for gradlew
+              run: chmod +x gradlew
+            - name: Build Skript
+              run: ./gradlew nightlyRelease
+            - name: Run test scripts
+              run: ./gradlew clean skriptTestJava8
```

Comment:
More an idea for the future, but I think it would be nice to have even more separation in our workflows, e.g. one workflow per MC version

Reply:
I think you can only have 6 running at a time. Don't want to overload GitHub and get in trouble or having it wait on eachother. I get it.

Reply:
Looked it up, it's actually 20 concurrent jobs [[1]](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits) [[2]](https://github.com/github/docs/discussions/1724#discussioncomment-146788). Regardless, I've commented it at https://github.com/SkriptLang/Skript/issues/3767#issuecomment-1192691981 because it's out of scope of this PR